### PR TITLE
jo 1.0

### PR DESCRIPTION
**Pull request by ear**
A small utility to [create JSON objects](https://github.com/jpmens/jo#jo) from the command line. Kind of an inverse to [jq](https://github.com/stedolan/jq#jq). :relaxed:

### CODE REVIEW

File: Library/Formula/jo.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  head do
+    url "https://github.com/jpmens/jo.git"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+  end
+
+  def install
+    system("autoreconf", "-i") if build.head?
```
Comment (mikemcquaid) [inline Library/Formula/jo.rb]: Can skip the parens here.

File: Library/Formula/jo.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+    system "./configure", "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "bin/jo", "success=true", "result=pass" == %({"success": true, "result": "pass"})
```
Comment (mikemcquaid) [inline Library/Formula/jo.rb]: This is failing: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/43198/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/test_jo/

File: Library/Formula/jo.rb (hunk 1 above)
Comment (ear) [inline Library/Formula/jo.rb]: Done.

File: Library/Formula/jo.rb (hunk 2 above)
Comment (ear) [inline Library/Formula/jo.rb]: It was indeed wrong. Reimplemented it using the same strategy as `jq`.

**Comment by ear** [comment]
What is the polite/right way to handle the commits here? I.e. should I squash them?

**Comment by dunn** [comment]
> I.e. should I squash them?

Yes, please!

**Comment by ear** [comment]
Done! Thanks @dunn

**Comment by dunn** [comment]
```
Error: 1 problem in 1 formula

==> audit problems
jo:
 * GitHub repository too new (<30 days old)
```
It's a pretty popular project, though, so I won't object if another maintainer merges this before the 30 days are up.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for new formulae is `jo 1.0 (new formula)`. Please complete the issue template in future PRs where this format is detailed in the linked [Contributing guidelines](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md). Thanks!

**Pull request closed without merging**